src/gadgets/scalar_gadgets/range.rs

```diff
@@ -350,7 +291,7 @@ mod tests {
     }
 
     #[test]
-    fn max_bound_test()  {
+    fn max_bound_test() {
```

Comment:
You can make the tests returns a `Result` instead of using `expect`.

Reply:
Done.